### PR TITLE
[WIP] Rackspace Migration Manager: Add draft of migration overview article

### LINKED ISSUE #1870: [MVP] Rackspace Migration Manager: Create draft overview article for migration from Rackspace to AWS
FAWS articles will not go live on the How-To landing page until the release currently scheduled for 3/7.

Something for Trent to then edit and add to.

### PR DESCRIPTION
**Pull request by cyrichardson**
Fixes #1870

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-140f08997d/how-to/)

**Comment by cyrichardson** [comment]
Sent Trent the link to this article to see if this is what he has in mind.

**Comment by stephamon** [comment]
May release

File: content/FAWS/migration-overview.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+type: article
+created_date: '2017-02-20'
+created_by: Catherine Richardson
+last_modified_date: '2017-02-20'
+last_modified_by: Catherine Richardson
+product: Fanatical Support for AWS
+product_url: fanatical-support-for-aws
+---
+
+Rackspace provides an easy self-service migration through the Fanatical Support&reg; migration control panel. The migration control panel was designed and built specifically for customer migrations from the Rackspace cloud to the Amazon Web Services (AWS) cloud. The control panel is intuitive and easy to navigate and provides a secure migration between Rackspace and AWS. With a little upfront planning on your part, the control panel steps you easily through the migration process.
```
Comment (trentgillaspie) [inline content/FAWS/migration-overview.md]: Fanatical Support migration control panel is now formally known as "Rackspace Migration Manager."

Comment (trentgillaspie) [inline content/FAWS/migration-overview.md]: Initially, the Rackspace Migration Manager has been built specifically for customer migrations from the Rackspace Public Cloud to Fanatical Support for Amazon Web Services (AWS). The migration manager is intuitive and easy to navigate, and provides a quick and secure migration between Rackspace and Fanatical Support for AWS. With a little planning on your part, the manager steps you easily through the migration process.

File: content/FAWS/migration-overview.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+The migration process leverages Rackspace and AWS APIs to securely move resources to the AWS cloud. The Rackspace source environment is not impacted as the migration to the AWS target environment takes place. (Make sure this is true - have seen some proposing to delete Rax cloud at end of migration.)
```
Comment (trentgillaspie) [inline content/FAWS/migration-overview.md]: The migration process leverages both Rackspace and AWS APIs to securely move resources to the Fanatical Support for AWS cloud. The Rackspace source environment is not impacted as the migration to the Fanatical Support for AWS target environment takes place.

File: content/FAWS/migration-overview.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+In the control panel, select your resources to be migrated after which recommendations for the migration are presented. Current and projected pricing are provided to help you make your selections. Finally, you schedule your migration. A "lift and shift" migration is done at the scheduled time.
```
Comment (trentgillaspie) [inline content/FAWS/migration-overview.md]: Using the Rackspace Migration Manager, simply select the resources you wish to migrate from the Rackspace Public Cloud to Fanatical Support for AWS. We will provide you with our recommended Fanatical Support for AWS environment setup, but you are free to customize configurations as well. Along the way, the migration manager will keep you informed of the current pricing of your selected resources and what you can expect for pricing after you have migrated to Fanatical Support for AWS. Finally, simply schedule the best time for your migration to take place, and track progress as things begin migrating from the migration dashboard within the manager tool.

Comment (trentgillaspie) [inline content/FAWS/migration-overview.md]: @cyrichardson these are all recommended language updates. Feel free to tweak things and let me know when you want me to take another pass to review. Thanks!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b132f42a13/how-to/)

**Comment by cyrichardson** [comment]
PR #2088 replaces this PR.

**Pull request closed without merging**